install.sh

```diff
@@ -112,7 +112,11 @@ tt_install() {
 
 	if [ ! -z "$queue" ]; then
 		echo
-		read -n 1 -p "Install missing dependencies? (Y/n)" CONFIRM
```

Comment:
Hmm, this feels a bit dangerous. What do you think about tt_install supporting a `-f` flag or something to bypass the confirmation?